service/src/main/java/com/fecklessweasel/service/objectmodel/FileMetadata.java

```diff
@@ -97,7 +97,7 @@ public static FileMetadata lookup(Connection sql, int fid) throws ServiceExcepti
      * @param course The course whose file info we want
      * @return A List of FileMetadata objects that represent each files info
      */
-    public static List<FileMetadata> lookUpCourseFiles(Connection sql, int course)
+    public static ArrayList<FileMetadata> lookUpCourseFiles(Connection sql, int course)
```

Comment:
same with univ/dept/class, this should be course.getFiles rather than Files.getFiles(course)